qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_mirror_to_rbd:
+    only Linux
+    start_vm = no
```

Comment:
only ceph
kill_vm = yes

Then we can pass the rbd conf options from outside

Reply:
Have update, thanks zhenchao.